### ISSUE TITLE
Signing & SBOM for controllers release artifacts

### ISSUE THREAD

**Issue by phoban01** [open-component-model/ocm-project#82]
## Description

In order to deliver a secure supply chain for ocm-controllers, we generating an SBOM and additionally sign all artifacts. 

Goreleaser provides support realizing this.

**Comment by Skarlso** [comment]
Alright, ocm-controller SBOMs are working. Same for replication controller is incoming.

**Comment by morri-son** [comment]
@Skarlso, what about creating OCM components for each of the controllers, or if it makes more sense (as single binaries are never installed, but only a complete set of binaries) one OCM component for the OCM controller in addition to the binaries? We can add the SBOM(s) as own artifact in parallel to the binaries.

**Comment by Skarlso** [comment]
Sure. That is a separate ticket though right? This about signing the binaries. We can then track progress on creating ocm components for all controllers once this is done if I understand correctly.

**Comment by morri-son** [comment]
yes, this will become for sure a new issue, this was just a question if it makes sense. Otherwise I as a PO just make it sense :-)

**Comment by Skarlso** [comment]
Nice. Let's do it.

**Comment by morri-son** [comment]
created https://github.com/open-component-model/ocm-project/issues/197

**Comment by Skarlso** [comment]
All done.
